src/test/java/pl/project13/maven/git/GitCommitIdMojoTest.java

```diff
@@ -27,6 +27,7 @@
 
 import java.io.File;
 import java.io.IOException;
+import java.nio.file.Paths;
 import java.util.Map;
```

Comment:
We don't really need JDK8 features and some companies are uber-slow in upgrading :-)
I agree that we should bump such dependency during a `2.2.0` :)